### ISSUE TITLE
Alt text is not descriptive [geo pages]

### ISSUE THREAD

**Issue by timngyn** [aws-amplify/docs#7051]
**Describe the content issue**:
Images listed below do not have descriptive/informative alt text. For markdown the alt text is located between the square brackets, `![alt text](...)`.

The list below isn't exhaustive since I found these images based on the word count for the alt text, but it should be a good starting point to make our alt text better for people who use screen readers.

Some resources to help write alt text:
- https://webaim.org/techniques/alttext/
- https://www.a11yproject.com/posts/are-you-making-these-five-mistakes-when-writing-alt-text/

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/build-a-backend/more-features/geo/configure-geofencing/index.mdx:
```
  194  
  195: Then click on the Add users button that displays a window to select the users by their username, email etc. that you intend to add to the Cognito group. Note: the user(s) added to the Cognito User Pool Group may need to be re-authenticated by signing out and signing in again. ![cognito group add users button](/images/cli/geo/geo-cognito-groups-2.png)
  196  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/prev/build-a-backend/more-features/geo/configure-geofencing/index.mdx:
```
  220  
  221: Then click on the Add users button that displays a window to select the users by their username, email etc. that you intend to add to the Cognito group. Note: the user(s) added to the Cognito User Pool Group may need to be re-authenticated by signing out and signing in again. ![cognito group add users button](/images/cli/geo/geo-cognito-groups-2.png)
  222  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/geo/ios/search/30_location_search_map.mdx:
```
  49  
  50: ![AMLMapView with AMLSearchBar](/images/ios-geocoder-search-box-map.png)
  51  

  75  
  76: ![AMLMapView featureImage view modifier](/images/ios-geocoder-custom-images.png)
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/geo/js/geofences.mdx:
```
  130  
  131: ![Geofence Create](/images/geofence-create.png)
  132  
  133: ![Geofence Edit](/images/geofence-edit.png)
  134  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/geo/js/google-migration.mdx:
```
  173  
  174: ![A map with a marker](/images/map-marker.png)
  175  

  295  
  296: ![A search box](/images/geocoder-custom-images.png)
  297  

  353  
  354: ![A search box](/images/geocoder-search-box.png)
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/geo/js/maps.mdx:
```
   94  
   95: ![A map centered on Vancouver](/images/display-map.png)
   96  

  148  
  149: ![A map with points](/images/display-map-with-points.png)
  150  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/geo/js/search.mdx:
```
  69  
  70: ![A search box](/images/geocoder-search-box.png)
  71  

  87  
  88: ![A search box](/images/geocoder-custom-images.png)
  89  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/geo/ios/search/30_location_search_map.mdx:
```
  46  
  47: ![AMLMapView with AMLSearchBar](/images/ios-geocoder-search-box-map.png)
  48  

  72  
  73: ![AMLMapView featureImage view modifier](/images/ios-geocoder-custom-images.png)
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/geo/js/geofences.mdx:
```
  131  
  132: ![Geofence Create](/images/geofence-create.png)
  133  
  134: ![Geofence Edit](/images/geofence-edit.png)
  135  
```

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/geo/js/google-migration.mdx:
```
  173  
  174: ![A map with a marker](/images/map-marker.png)
  175  

  295  
  296: ![A search box](/images/geocoder-custom-images.png)
  297  

  353  
  354: ![A search box](/images/geocoder-search-box.png)
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/geo/js/maps.mdx:
```
   94  
   95: ![A map centered on Vancouver](/images/display-map.png)
   96  

  148  
  149: ![A map with points](/images/display-map-with-points.png)
  150  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/geo/js/search.mdx:
```
  69  
  70: ![A search box](/images/geocoder-search-box.png)
  71  

  89  
  90: ![A search box](/images/geocoder-custom-images.png)
  91  
```
